geostore/tiles/mixins.py

```diff
@@ -46,23 +46,27 @@ def get_maxzoom(self):
                 for l in self.layers
             ]))
 
-    def get_attribution(self):
-        return ','.join(set(
-            [
-                l.layer_settings_with_default('metadata', 'attribution')
-                for l in self.layers if l.layer_settings_with_default('metadata', 'attribution')
-            ])) or None
+    def _join_group_settings_link(self, layers, *args):
+        return ','.join(set([
+            a if 'link' not in a else
+            '<a href="{0}"/>{1}</a>'.format(a['link'].replace('"', '&quot;'), a['name'].replace('"', '&quot;'))
```

Comment:
- Isn't `django.utils.encoding.escape_uri_path()` better than `a['link'].replace('"', '&quot;')` given that html entities shouldn't be used in a path?
- Isn't `django.utils.html.escape()` better than `a['name'].replace('"', '&quot;')`?

Reply:
You're right, fixed.